DiabloII.Application/Controllers/SuggestionsController.cs

```diff
@@ -34,8 +39,20 @@ public SuggestionsController(ISuggestionReader reader, ISuggestionCommandHandler
         [Route("suggestions")]
         [HttpGet]
         [ProducesResponseType(typeof(IReadOnlyCollection<SuggestionDto>), StatusCodes.Status200OK)]
-        public ActionResult<IReadOnlyCollection<SuggestionDto>> GetAll() =>
-            GetAll(_reader, _mapper);
```

Comment:
it could be great to get this level of abstraction, with the possibility to add hal service as parameter